tgui/packages/tgui/interfaces/ChemReactionChamber.tsx

```diff
@@ -134,7 +134,7 @@ export const ChemReactionChamber = (props) => {
               title="Settings"
               height="220px"
               maxHeight="220px"
-              overflow="hidden"
+              overflow="scroll"
```

Comment:
there's a prop on section just for this. use `scrollable` please

Reply:
Nope it doesn't work.


https://github.com/user-attachments/assets/34cb234e-48ea-4777-80a2-f7c69a1f69e9

Reply:
But how would the section know when to display the scroll bar if we are just goanna keep expanding it's height.

The height prop tells the section if it's contents exceed that value start scrolling, but fill is just goanna keep expanding the height?

None the less I'll give it a try

Reply:
did that change how the ui looks at all? you can use `grow={2}`, `grow={3}` etc to get a proportional ui without having to resort to pixel measurements

Reply:
The UI looks the same because it still has its height property set. Only the maxHeight property was replaced with fill.

We still need pixel measurements because of that empty space in the center  of each section which decides how many reagents to display before scrolling is needed